From b9521409c1c4b5527d0861af04aac2b28f34487c Mon Sep 17 00:00:00 2001
From: Hariprasad Kelam <hkelam@marvell.com>
Date: Fri, 15 May 2020 14:20:10 +0530
Subject: [PATCH 04/10] octeontx2-af: Increase mbox response timeout to 3
 seconds

commit df0f9a703ba6a633789a38327f2327b409e83247 from
git@git.assembla.com:cavium/WindRiver.linux.git

Upon receiving mbox request from PF to configure speed, fec, higig
AF sends commands to firmware and waits for max response time
which is 2.2 seconds.

In few scenarios firmware is taking more time to process the commands
and PF is not getting mbox reply as firmware command takes more time
than mbox response timeout which is 2 seconds. So increase the same
to 3 seconds to accommodate above case.

Change-Id: Ia114f01ae752d32da00f63cdf428aa4549da19f0
Signed-off-by: Hariprasad Kelam <hkelam@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/28642
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/mbox.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/mbox.h b/drivers/net/ethernet/marvell/octeontx2/af/mbox.h
index b94777ad2e36..b68e80150f7c 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/mbox.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/mbox.h
@@ -36,7 +36,7 @@
 
 #define INTR_MASK(pfvfs) ((pfvfs < 64) ? (BIT_ULL(pfvfs) - 1) : (~0ull))
 
-#define MBOX_RSP_TIMEOUT	2000 /* Time(ms) to wait for mbox response */
+#define MBOX_RSP_TIMEOUT	3000 /* Time(ms) to wait for mbox response */
 
 #define MBOX_MSG_ALIGN		16  /* Align mbox msg start to 16bytes */
 
-- 
2.17.1

